src/components/projects/view/application-tile.svelte

```diff
@@ -1,107 +1,149 @@
 <script>
-  import { onMount, createEventDispatcher } from 'svelte';
-  import AccordionSection from 'ui/accordion-section.svelte';
-  import Button from 'ui/button.svelte';
-  import CopyButton from '@/components/projects/view/copy-button.svelte';
+  import { onMount, createEventDispatcher, getContext } from 'svelte';
+  import { stores } from '@sapper/app';
+  import { AccordionSection, Button, Popover } from 'attractions';
+  import { PopoverPositions } from 'attractions/popover';
+  import { snackbarContextKey } from 'attractions/snackbar';
+  import CopyButton from '@/components/common/copy-button.svelte';
+  import Stars from '@/components/common/stars.svelte';
+  import ViewReportsDialog from '@/components/projects/view/view-reports-dialog.svelte';
   import ApplicationStatuses from '@/constants/backend/application-statuses.js';
   import * as api from '@/utils/api.js';
 
+  const { session } = stores();
+
   export let activity;
   export let application;
-  export let handlePanelOpen;
-  let report = null;
 
+  let report = null;
   onMount(() => {
     report = api.get(
       `/projects/${activity.project}/activities/${activity.id}`
       + `/applications/${application.id}/report_info`,
-    ).then(resp => resp.json());
+    ).then(resp => resp.json()).catch(console.error);
```

Comment:
Note that adding the `.catch` will make the `report` promise resolve, causing `reportData` in the template to be `undefined` in case the API call fails (because it is what `console.error` returns)

Reply:
Also, `onMount` can take an `async` function, but that doesn't mean other code won't run until it resolves.

Reply:
Just switched it to `api.json` and removed the catch. Seems to be working fine when the server rejects that endpoint